demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -317,6 +317,41 @@ def generate_readmes_for_new_content_pack(self, is_contribution=False):
                     if file_name.startswith('playbook') and file_name.endswith('.yml'):
                         self.generate_readme_for_pack_content_item(file)
 
+    def rearranging_before_conversion(self) -> None:
+        """
+        This method was written in order to fix a bug that exists in the server,
+        which incorrectly maps indicatorfield to incidentfield directory.
+        If this happened, this method will map the indicatorfield files to a separate folder called - indicatorfiled.
+        """
+        unpacked_contribution_dirs = get_child_directories(self.pack_dir_path)
+
+        for unpacked_contribution_dir in unpacked_contribution_dirs:
+
+            dir_name = os.path.basename(unpacked_contribution_dir)
+
+            # incidentfield directory may contain indicator-fields files
+            if dir_name == FileType.INCIDENT_FIELD.value:
+
+                dst_dir_path = None
+                is_first_file = True
+                src_path = os.path.join(self.pack_dir_path, dir_name)
+
+                for file in os.listdir(src_path):
+
+                    if file.startswith(FileType.INDICATOR_FIELD.value):
+                        # At first time, create another dir for all indicator-fields files and move them there
+                        if is_first_file:
+                            dst_dir_name = FileType.INDICATOR_FIELD.value
+                            dst_dir_path = os.path.join(self.pack_dir_path, dst_dir_name)
+                            os.makedirs(dst_dir_path)
+                            is_first_file = False
```

Comment:
Lazy loading, nice.
No need to save `dst_dir_name` var, and `is_first_file` isn't needed.

Reply:
Done